src/scripts/main.js

```diff
@@ -1,17 +1,41 @@
 'use strict';
 
 const pushNotification = (posTop, posRight, title, description, type) => {
-  // write code here
+  document.body.insertAdjacentHTML('afterbegin', `<div class="notification">
+    <h2 class="${title}">${title}</h2><p>${description}</p></div>`);
```

Comment:
```suggestion
  document.body.insertAdjacentHTML('afterbegin', `
    <div class="notification">
      <h2 class="${title}">${title}</h2>
      <p>${description}</p>
    </div>
  `);
```
Форматируй html тут так, чтобы его было удобно читать

Reply:
Володимир залишив слушний коментар. Варто форматувати саме так, як він навів у прикладі